javascript/javascript1/week2/flight-booking-fullname-function.js

```diff
@@ -0,0 +1,31 @@
+// Function getFullaname() returns full name with/without Formal
+
+
+function getFullname(firstname, surname, useFormalName) {
+    if (!firstname) return new Error("Firstname  must not be null");
```

Comment:
Instead of return an error, throw it `throw new Error('..');`
Check if firstname and surname are empty strings like this `""` or maybe long-empty string `"    "`
- check this [trim](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim)